src/entities/Votes/utils.ts

```diff
@@ -159,3 +162,22 @@ export function abbreviateNumber(vp: number) {
 
   return scaled.toFixed(1) + suffix
 }
+
+export async function getProposalScores(proposal: ProposalAttributes, addresses: string[]) {
+  const result = {} as Record<string, number>
```

Comment:
Maybe you can move `Record<string, number>` to a type and reuse it in this function.
`type Scores = Record<string, number>`